dataland-frontend/src/components/resources/frameworkDataSearch/sfdr/SfdrPanel.vue

```diff
@@ -5,6 +5,7 @@
   </div>
   <div v-if="kpiDataObjects.length && !waitingForData">
     <CompanyDataTable
+      data-test="sfdrCompanyTable"
```

Comment:
As I have mentioned in another comment, the name of this component presumes a generic character of it, while in fact, `CompanyDataTable` is only being used for the sfdr framework view page.
So let's rename it, and let's also change the "data-test" marker here accordingly.

Reply:
Would `SfdrDataTable` be OK?

Reply:
Renaming it now

Reply:
Renamed to SfdrDataTable

Reply:
data-test marker still has the old name

Reply:
👍